pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -223,7 +292,89 @@ func (r *rbac) createResources(ctx context.Context) error {
 			return err
 		}
 
-		if err := r.ensurePipelinesSCCRoleBinding(ctx, sa); err != nil {
+		// kind of role in the namespace, ClusterRole by default
+		roleKind := "ClusterRole"
+
+		// If "operator.tekton.dev/scc" exists in the namespace, then bind
+		// that SCC to the SA
+		nsAnnotations := n.GetAnnotations()
+		namespaceSCC := nsAnnotations[v1alpha1.NamespaceSCCAnnotation]
+		if namespaceSCC != "" {
+			logger.Infof("Namespace: %s has requested SCC: %s", n.GetName(), namespaceSCC)
+
+			// We don't want to use ClusterRole now
+			roleKind = "Role"
+
+			// Make sure that SCC exists
+			if err := r.verifySCCExists(ctx, namespaceSCC); err != nil {
+				logger.Error(err)
+				failureEvent := corev1.Event{
+					ObjectMeta: metav1.ObjectMeta{
+						GenerateName:    "pipelines-scc-failure-",
+						Namespace:       n.GetName(),
+						OwnerReferences: []metav1.OwnerReference{r.ownerRef},
+					},
+					EventTime:           metav1.NewMicroTime(time.Now()),
+					Reason:              "RequestedSCCNotFound",
+					Type:                "Warning",
+					Action:              "DefaultSCCUSed",
+					Message:             fmt.Sprintf("SCC '%s' requested in annotation '%s' but could not find SCC, OpenShift Pipelines workloads will not run in this namespace", namespaceSCC, v1alpha1.NamespaceSCCAnnotation),
```

Comment:
Workloads can run, but with the openshift "default" scc and stuff right ? or the cluster-wide max ?

Reply:
yep, changed the message to
```
fmt.Sprintf("SCC '%s' requested in annotation '%s' not found, SCC not updated in the namespace", scc, namespaceSCCAnnotation)
```